### DISCUSSION TITLE
Is there any way to calculate collision using the real size of the draggable element?

### DISCUSSION THREAD

**Discussion by yf-hk** [clauderic/dnd-kit#233] [Q&A]
For example, if you set `transform: scale(2)` on the overlay, currently collision detection would be still caculated based on the original size. It's inaccurate now. Is there any configuration or workaround that can make it work?
For example, I'm dragging a photo to a canvas with a zoomed size:
![image](https://user-images.githubusercontent.com/203980/115807274-e7ccdd80-a41a-11eb-881e-151d9f8b045f.png)
Because it's larger than the original size so when it drags over dnd-kit cannot detect collisions correctly. (When isOver === true the background turns red)
![image](https://user-images.githubusercontent.com/203980/115807231-d1268680-a41a-11eb-8991-4b711f1cd845.png)
It can only trigger the `over` event with the original size

**Comment by clauderic** [comment]
This use-case isn't currently well-supported. There's a number of changes that would need to happen in the library to support this. 

Changes to the `transform` property of the `DragOverlay` children would never trigger resize events, so there's no way for it to know about this.

We could however introduce a `scale` property on `DragOverlay` to support this use-case and update the collision detection `width` and `height` based on that scale.

You could also try to achieve this with modifiers, but again modifiers don't _currently_ affect the `width` and `height` of the collision rect, that's a bug.
